test/rules/always_put_control_body_on_new_line.dart

```diff
@@ -19,6 +19,13 @@ testIfElse() {
     return; // OK
   else return; // LINT
 
+  if (false) {
+  }
+  else if (false) { // OK
```

Comment:
> I checked with @sethladd, and as far as I can tell there are no plans to have a second formatter.

Allow me to be even more specific: There are no plans to ask or force our end-user developers to use a different formatter. dartfmt is sufficient for our end-user needs, and the Flutter SDK style isn't required for end-user Flutter developers.

I don't know what the plans are from the Flutter SDK authors for their own formatting needs.